src/foam/u2/view/MultiChoiceView.js

```diff
@@ -79,16 +80,17 @@ foam.CLASS({
         return [];
       },
       adapt: function(_, n) {
-        if ( ! Array.isArray(n) ) throw new Error("Please submit an array to choices in the MultiChoiceView");
+        if ( ! Array.isArray(n) ) throw new Error('Please submit an array to choices in the MultiChoiceView');
 
-        if ( n.length > 0 ){
+        if ( n.length > 0 ) {
+          n = n.sort();
```

Comment:
apart of style commit - but this is what is used to alphebetically set the choices of the minMax